3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -179,6 +182,7 @@ object EnrichmentManager {
           ("ti_ca"   , (ME.toTsvSafe, "ti_category")),
           ("ti_pr"   , (CU.stringToDoublelike, "ti_price")),
           ("ti_qu"   , (ME.toTsvSafe, "ti_quantity")),
+          ("ti_cu"   , (CU.stringToDoublelike, "ti_currency")),
```

Comment:
Currency is a string like GBP, it's not something that should be parsed to a Double.